app/controllers/time_tracker_controller.rb

```diff
@@ -12,6 +12,7 @@ def start
 
     # Go to start page if timer session is not valid
     unless @current_timer_session.valid?
+      @current_timer_session.issues << Issue.find(timer_params[:issue_ids]) if timer_params[:issue_ids].present?
```

Comment:
Attention: `timer_params[:issue_ids]` might be an array. Does `<<` work with arrays?

Reply:
Good point! I will update my spec to account for the use of multiple issue_ids.